src/classes/Foo.cls

```diff
@@ -0,0 +1,6 @@
+public without sharing class Foo {
+    void foo() {
+        HttpRequest req = new HttpRequest();
+        req.setEndpoint('http://localhost:com');
```

Comment:
Please use named credentials so that endpoints and authorizations are managed automatically by Salesforce.